### PR TITLE
2128 split drive not handling relative paths

### PR DESCRIPTION
**Pull request by mnowster**
Should fix https://github.com/docker/compose/issues/2128

### CODE REVIEW

**Comment by dnephin** [comment]
LGTM

File: compose/config/config.py [hunk 1]
```diff
@@ -485,7 +485,6 @@ def resolve_volume_paths(service_dict, working_dir=None):
 
 def resolve_volume_path(volume, working_dir, service_name):
     container_path, host_path = split_path_mapping(volume)
-    container_path = os.path.expanduser(container_path)
```
Comment (dnephin) [inline compose/config/config.py]: Oh wait, why is this removed?

Comment (mnowster) [inline compose/config/config.py]: Because `container_path` should never be relative, so we don't need to call `expanduser` on it. @aanand spotted it last time when I was working through understanding the paths situation.

Comment (dnephin) [inline compose/config/config.py]: I guess it would expand to the wrong path, that's true.  We added this in 1.4.0, so we should probably call this out in the release notes as a bug fix.
